### DISCUSSION TITLE
Uploader : upload files to jwt authenticated url

### DISCUSSION THREAD

**Discussion by PeteMaxy** [rsuite/rsuite#3292] [Q&A]
I am new to rsuite. I would like to know how to upload files to jwt authenticated url. What should I do?

![1](https://github.com/rsuite/rsuite/assets/20745271/a75c55ab-3803-4b36-b0d7-947ef930b9c0)

**Comment by SevenOutman** [comment]
Does "jwt authenticated url" mean you need to append the jwt as a search param of the url? That way you could simply pass a concat string to `action`

```jsx
<Uploader
  action={`/endpoint?jwt=${token}`}
/>
```

---

BTW, I'm converting this issue to a discussion which is for Q&A.
